docker-compose.desktop.yml

```diff
@@ -35,19 +39,17 @@ services:
 
         https:// {
           log
-          reverse_proxy * firezone:13000
-          tls internal {
-            on_demand
-          }
+          reverse_proxy * 172.25.0.100:${PHOENIX_PORT:-13000}
+          ${TLS_OPTS:-}
         }
         EOF
     deploy:
       <<: *default-deploy
 
   firezone:
-    image: firezone/firezone
+    image: firezone/firezone:${VERSION:-latest}
```

Comment:
I made it a safe default so that behavior would not change if you have a more recent docker-compose file while having an old `.env`.

Reply:
👍